src/sentry/options/defaults.py

```diff
@@ -279,14 +279,30 @@
 )
 
 # API
-# Killswitch for apis to work with id or slug as path parameters
+# Killswitch for GA endpoints to work with id or slug as path parameters
```

Comment:
i would refrain from calling these "Killswitches" as those are a specific concept in our codebase: https://github.com/getsentry/sentry/blob/3a72bb65fc70d1fcad55a6f5965048d2390e6699/src/sentry/killswitches.py#L278